### ISSUE TITLE
WCAG2Mobile: Draft feedback - 1.4.12 Text Spacing

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#656]
(Mary Jo) 

- MATF removed the phrase "implemented using markup languages" and now it reads incorrectly. It isn't the content that would support those text style properties, it would have to be the platform that supports them as well as user modification of them...and then the content would be required to abide by those settings. Word replacement should instead be something like "content implemented for platforms"
- MATF deleted the 3 WCAG2ICT notes (Notes 1 and 2 were about markup languages), so since they removed the markup language content, so those makes sense.) However, MATF didn't include WCAG2ICT Note 3 which I think still makes sense in the mobile space. We should suggest they include it.
- Last MATF note does something normative - says the SC wouldn't apply where the platform doesn't include a way for users to override text style properties. Additionally, this could be misinterpreted that if one of the listed properties don't have a way for the user to modify, and does support others to be modified, this SC wouldn't apply. I don't think that was the intent.
- IMO, an issue should be opened against WCAG on this - they should have a more generic exception for when a user agent doesn't provide this capability. C

(Chris )

- If a mobile platform does not include a way for users to override text style properties, this success criterion is not applicable. is introduced on https://www.w3.org/TR/wcag2mobile-22/#success-criterion-1-4-12-text-spacing and is not specifically mentioned on https://www.w3.org/TR/wcag2ict/#text-spacing . 
- If different, this should be highlighted as per what they are stating currently , it applies directly as written, where they are adding a note , so it doesn't necessarily apply as written as they are changing what is written per mobile vs. wcag2ict.
